README.md

```diff
@@ -1,62 +1,41 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My To-Do List
 
-> Description the project.
+> This is a project undertaken in week two of module two to test our abilities to develop a functional application using Webpack.
```

Comment:
- [ ] **Optional:** This is a good description. However, it can be improved by giving a description of the functionalities of the app in this section.
In future projects, consider implementing this. 🚁

Reply:
- [OPTIONAL] As required by the previous, Please, add a meaningful  `Project description`. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how.

- [OPTIONAL] Kindly, it would be nice to add explicit steps for cloning your app, as this will be of help to non-technical users. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd..`, etc.

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.